src/interface/events.ts

```diff
@@ -16,6 +17,7 @@ export enum InterfaceEventName {
   NAVBAR_SEARCH_EXITED = 'Navbar Search Exited',
   NAVBAR_SEARCH_SELECTED = 'Navbar Search Selected',
   RISK_CHECKED = 'Risk Checked',
+  SELECT_CHAIN_INITIATED = 'Select Chain Initiated',
```

Comment:
logged when the user selects a chain in the chain selector

Reply:
do we also want event for network switcher dropdown clicked? per Linear ticket

Reply:
Can we use an element click event for this one?